python/train.py

```diff
@@ -18,61 +18,70 @@
 from __future__ import print_function
 
 import math
+from ffcc import model
+from ffcc import ops
+from ffcc import io
+from ffcc import input as ffcc_input
+import os
+import json
+from tensorboard.plugins.hparams import api as hp
+import tensorflow as tf
 
-from . import model
-from . import ops
-import input as ffcc_input
-import tensorflow.compat.v1 as tf
 
-# This directory stores the TFRecord for training.
-DATA_DIR = ''
+os.environ['CUDA_VISIBLE_DEVICES'] = '-1'
+
+
+# This directory stores the images and metadata for training.
+DATA_DIR = '../data/shi_gehler/preprocessed/GehlerShi'
+
+# Project name
+PROJECT_NAME = 'GehlerShi'
 
 # Model dir for checkpoints, etc.
-MODEL_DIR = ''
+MODEL_DIR = './GehlerShi_model/'
 
 # The number of epochs for training.
 NUM_EPOCHS = 150
 
+# Number of testing fold in the 3-fold cross validation
+TEST_FOLD = 1 # if TEST_FOLD=0, the code will train three models, each of
```

Comment:
Comments should go before the code they comment on (or at the end of the line, but not continuing to a new line in that case).